modules/dnn/src/op_inf_engine.cpp

```diff
@@ -622,14 +622,20 @@ void InfEngineBackendNet::init(int targetId)
 
 #endif  // IE < R5
 
-static std::map<InferenceEngine::TargetDevice, InferenceEngine::InferenceEnginePluginPtr> sharedPlugins;
+static std::map<InferenceEngine::TargetDevice, InferenceEngine::InferenceEnginePluginPtr>& getSharedPlugins()
+{
+    static std::map<InferenceEngine::TargetDevice, InferenceEngine::InferenceEnginePluginPtr> sharedPlugins;
+    return sharedPlugins;
+}
 
 void InfEngineBackendNet::initPlugin(InferenceEngine::ICNNNetwork& net)
 {
     CV_Assert(!isInitialized());
 
     try
     {
+        AutoLock lock(getInitializationMutex());
```

Comment:
perhaps const if not modified.

Reply:
@shahzadlone, thanks for review! Could you please clarify it a bit?

Reply:
if lock isn't being modified could it perhaps be made AutoLock const?

Reply:
This change doesn't fix (or change) anything in current code.

@shahzadlone Please report about real issues only.

Avoid reporting about "const" of local variables. They are usually not required and can be safely omitted (compilers are smart enough).
It make sense to report about "const" in parameters or about "const methods".